### PR TITLE
Feature/opcache flush

### PR DESCRIPTION
**Pull request by joverthegrey**
SA-7317 and travis fix

### CODE REVIEW

**Comment by joverthegrey** [comment]
@M52 @BaukePalstra  kunnen jullie de pr eeven reviewen ?

File: src/class-cache-flusher.php [hunk 1]
```diff
@@ -50,4 +56,16 @@ function flush_domain() {
...
     }
+
+    /**
+     * Flush the OpCache if enabled
+     */
+    private function flush_opcache()
+    {
+        $opcache_status = opcache_get_status();
+
+        if ($opcache_status && is_array($opcache_status) && $opcache_status['opcache_enabled']) {
```
Comment (M52) [inline src/class-cache-flusher.php:67]: opcache_get_status() can return an Array or false. If it is false, the you can't get the index 'opcache_enabled'. Should be:

```
if ($opcache_status && is_array($opcache_status)) {
  if ($opcache_status['opcache_enabled']) {
    // do something
  }
}
```

File: src/class-options.php [hunk 2]
```diff
@@ -25,14 +25,28 @@ class Options {
...
+
+    /**
+     * Wrapper around getenv()
+     * When not found with getenv() also check $_ENV and $_SERVER.
+     *
+     * @param $name
+     * @return array|false|mixed|string
+     */
+    private static function env($name) {
+        return (getenv($name) ?:
```
Comment (M52) [inline src/class-options.php:47]: Minor inconsistency, using `getenv` while using `self::env` in all other instances when reading environment variables.

**Review by M52 — COMMENTED** [review]
Looks good, see comment in flush_opcache() function.

File: src/class-cache-flusher.php (hunk 1 above)
Comment (joverthegrey) [reply src/class-cache-flusher.php:67]: As php checks an and clause from left to right, this if is implicit in the third part of the and.
If $opcache_status == false it fails the and clause and never reaches the array check

**Review by M52 — APPROVED** [review]
Approved.

**Pull request merged**